core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -58,15 +58,6 @@ private[spark] class TaskSetManager(
 
   val conf = sched.sc.conf
 
-  /*
-   * Sometimes if an executor is dead or in an otherwise invalid state, the driver
-   * does not realize right away leading to repeated task failures. If enabled,
-   * this temporarily prevents a task from re-launching on an executor where
-   * it just failed.
-   */
-  private val EXECUTOR_TASK_BLACKLIST_TIMEOUT =
-    conf.getLong("spark.scheduler.executorTaskBlacklistTime", 0L)
```

Comment:
we need to be sure to still support this conf name.  In fact I think you can just stick w/ this instead of switching to "spark.scheduler.blacklist.timeout", right?  perhaps that conf should only be for strategies which do per-task blacklisting, but I feel like it could be for all cases.

(In any case, I think we defer these exact naming decisions till we're closer on the implementation, just mentioning it now while I'm noticing it.)